pom.xml

```diff
@@ -36,7 +36,7 @@
         <!-- Maven and Surefire plugins -->
         <maven-surefire-plugin.version>3.3.1</maven-surefire-plugin.version>
         <maven-compiler-plugin.version>3.13.0</maven-compiler-plugin.version>
-        <spring-web-version>6.1.12</spring-web-version>
+        <spring-web-version>6.1.15</spring-web-version>
```

Comment:
Are you able to try bumping up the spring boot version to the latest release today?
* Spring Boot 3.3.6

If you look back at the pom.xml commit ef42f522f09b480f16c4750eb7174a2c28ed2579 `Author: jaddison-ch Date: 22/08/2024, 09:01`, then I think we can try to remove the `spring-security.version` stuff along with the `spring-web` exclusion & override version.

See https://docs.spring.io/spring-boot/appendix/dependency-versions/coordinates.html
* spring-security-config | 6.3.5
* spring-security-core | 6.3.5
* spring-security-web | 6.3.5